Popup

asnganje opened the pull request:
- Created dynamic page
- Included javascript file to store information 
- Added popup window

asnganje committed:
added html tags for modal

asnganje committed:
modified, styles, modal, and index html file

asnganje committed:
fixing linter errors

asnganje committed:
fixed styles.css linters

asnganje committed:
fixed linter errors

asnganje committed:
linter errors fixing

asnganje committed:
linter errors fixed

asnganje committed:
fixed linter errors

asnganje committed:
fixed linter errors

asnganje committed:
fixed linter errors

danmainah reviewed (changes requested):
Hi @asnganje , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has commits from one person and its a group project, kindy ensure everybody contributes to the project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

asnganje committed:
modified styles.css

asnganje committed:
modified styles and modal files

asnganje committed:
fixed linter errors in styles.css

roshan-bajgain reviewed (changes requested):
Hi @asnganje, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has done individually and this is a pair programming project. Kindly inform student success. I cannot do anything I am sorry. I need to follow the senior code reviewer and I got the same suggestion. Kindly inform student success so you can move forward

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

LuisDavidRodriguez committed:
Add parent background, color transparent

asnganje committed:
modified stles.css

asnganje committed:
fixed linter errors

mugishasam123 commented on index.html:
```diff
@@ -54,40 +57,18 @@ <h2 class="support-text">I’am a Software Developer</h2>
     <section class="project-portfolio">
         <div class="mediacss-grid">
             <div class="gitem1">
-                <button class="buttongrid1" type="button">See Project</button>
+                <button class="buttongrid1 button2" type="button">See Project</button>
             </div>
```
- [x] Please try to stick to Figma design as much as possible to make your work look professional .so you can try to leverage the positioning of items in the popup modal to match the design (fonts,colors, margins,padding,etc)

your design:
![image](https://user-images.githubusercontent.com/90524466/180284211-49751801-15d5-43c9-b98c-2baa8b272b09.png)

Figma design:
![image](https://user-images.githubusercontent.com/90524466/180284477-b6c7e77e-9ee7-4e42-9e54-700e91fcc2d0.png)

mugishasam123 reviewed (changes requested):
Hi @asnganje  👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![alt](https://media3.giphy.com/media/SDunHzTgSZbSuQy1EP/giphy.gif)
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors 💯 
- Github workflow was properly used 👍 , keep rocking 🚀 


## Required Changes ♻️
_Check the comments under the review._

- [x] Please try to fix the see project buttons in other projects' cards so that you can make your portfolio more engaging and professional. it seems they are not showing up and working .you got this 👍 

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

asnganje committed:
fixed all see project buttons and close button position

asnganje committed:
modified styles.css

lucascomamala committed:
Fix modal design

- Changed the design of the popup modal to stick more closely to the Figma template
- Added See Project buttons for all projects

asnganje committed:
fixed linter errors

Ismailco commented on modal.js:
```diff
@@ -0,0 +1,178 @@
+const myMultipost = document.getElementById('portfolioModal');
+
+const myRecentWorks = [
```
- [ ] [OPTIONAL] You can add more projects to the list so your project's popup is different when clicking on different projects

Ismailco commented on index.html:
```diff
@@ -54,158 +57,76 @@ <h2 class="support-text">I’am a Software Developer</h2>
                 <ul class="gitem4-s">
-                <li>html</li>
-                <li>bootstrap</li>
-                <li>Ruby</li>
+                  <li>html</li>
+                  <li>bootstrap</li>
+                  <li>Ruby</li>
                 </ul>
+                <button class="buttongrid1 button2" type="button">See Project</button>
             </div>
             <div class="gitem5">
                 <h4 class="gitem5-h">Data Dashboard Healthcare</h4>
                 <p class="gitem5-p">A daily selection of privately personalized
                 reads; no accounts or sign-ups required. has been the industry's standard</p>
                 <ul class="gitem5-s">
-                <li>html</li>
-                <li>bootstrap</li>
-                <li>Ruby</li>
+                  <li>html</li>
+                  <li>bootstrap</li>
+                  <li>Ruby</li>
                 </ul>
+                <button class="buttongrid1 button2" type="button">See Project</button>
             </div>
             <div class="gitem6">
                 <h4 class="gitem6-h">Website Protfolio</h4>
                 <p class="gitem6-p">A daily selection of privately personalized
                 reads; no accounts or sign-ups required. has been the industry's standard</p>
                 <ul class="gitem6-s">
-                <li>html</li>
-                <li>bootstrap</li>
-                <li>Ruby</li>
+                  <li>html</li>
+                  <li>bootstrap</li>
+                  <li>Ruby</li>
                 </ul>
+                <button class="buttongrid1 button2" type="button">See Project</button>
             </div>
         </div>
         <div class="grid-container2">
```
- [ ] [OPTIONAL] The project section should be created dynamically using JavaScript, kindly implement this so your project will be complete

Ismailco reviewed (approved):
### Hi @asnganje, @LuisDavidRodriguez,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![cr-approve](https://user-images.githubusercontent.com/25227022/165483662-7b06c812-4cc0-40b9-a84c-4729d4cf10c3.gif)


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
